Formula/git-delta.rb

```diff
@@ -14,12 +14,15 @@ class GitDelta < Formula
   end
 
   depends_on "rust" => :build
-  uses_from_macos "llvm"
 
   conflicts_with "delta", :because => "both install a `delta` binary"
 
   def install
-    system "cargo", "install", "--root", prefix, "--path", "."
+    # `cargo install` produces binary that have problems running on Catalina
+    # rubocop:disable FormulaAudit/Text
+    system "cargo", "build", "--release", "--locked"
+    # rubocop:enable FormulaAudit/Text
+    bin.install "target/release/delta"
```

Comment:
We prefer `cargo install` to manually installing files into `bin`, because this leaves upstream responsible for what they want to distribute or not (several binaries, doc, examples, etc.).

Reply:
Please see: https://github.com/dandavison/delta/issues/24

Reply:
This is working around a bug in the release. It was reported 7 hours ago, so let's give upstream time to fix their bug, cut a new release (they seem to move fast), and we'll ship that.